tmt/utils/__init__.py

```diff
@@ -6661,3 +6664,99 @@ def render_rst(text: str, logger: Logger) -> str:
     document.walkabout(visitor)
 
     return visitor.rendered
+
+
+def import_jira() -> None:
+    """ Import Python Jira library """
+    try:
+        global JIRA
+        from jira import JIRA
+    except ImportError:
+        raise GeneralError(
+            "Install 'tmt+link-jira' to use the Jira linking.")
+
+
+def jira_link(
```

Comment:
Since we are slowly splitting `tmt.utils` into smaller modules by topic, please consider putting this new code into `tmt.utils.jira` from the beginning.